.github/workflows/ci-docs-tests.yml

```diff
@@ -60,12 +60,25 @@ jobs:
         poetry-version: ${{ env.POETRY_VERSION }}
 
     - name: Load Poetry Cached Libraries ⬇
+      id: cache-poetry
       uses: actions/cache@v2
       with:
-        path: ~/.cache/pypoetry/virtualenvs
+        path: .venv
         key: ${{ runner.os }}-poetry-${{ env.POETRY_VERSION }}-${{ env.DEFAULT_PYTHON_VERSION }}-${{ hashFiles('**/poetry.lock') }}-${{ secrets.POETRY_CACHE_VERSION }}
         restore-keys: ${{ runner.os }}-poetry-${{ env.DEFAULT_PYTHON_VERSION }}
 
+    - name: Clear Poetry cache
+      if: steps.cache-poetry.outputs.cache-hit == 'true' && contains(github.event.pull_request.labels.*.name, 'tools:clear-poetry-cache-docs-tests')
```

Comment:
do we document PR labels like `tools:clear-poetry-cache-docs-tests` somewhere? I haven't created the label yet on github (same for other new labels introduced in this PR)

Reply:
This [notion page](https://www.notion.so/rasa/PR-deployments-f95cc7feee70453388e7d306dc75ee79) is the closest I can find

Reply:
🤔 I guess I'll just create the labels in GitHub for now, and describe them here